src/Features/Dashboard/Reducer/reducer/index.js

```diff
@@ -0,0 +1,86 @@
+const initialData = {
+  cart: [],
+  isDrawerOpen: false,
+};
+export const addtocartReducers = (state = initialData, action) => {
+  const updatedCart = [];
+
+  switch (action.type) {
+    case 'OPEN_DRAWER':
+      return { ...state, isDrawerOpen: true };
+    case 'CLOSE_DRAWER':
+      return { ...state, isDrawerOpen: false };
+    case 'ADD_TOCART':
+      // eslint-disable-next-line no-case-declarations
+      const { id, name, price, img, qty } = action.payload;
+      console.log(action.payload, 'id');
+      // yaha per logic likho agr id phele s car ki array m he to uska qty increase kare
+      // eslint-disable-next-line no-case-declarations
+      const filter = state.cart.filter((elem) => elem.id === id, console.log('ele', state.cart));
```

Comment:
remove console.log and make code readable, code should be readable by the human not only machine